packages/langium/src/services.ts

```diff
@@ -45,6 +45,7 @@ import type { FileSystemProvider } from './workspace/file-system-provider';
 import type { IndexManager } from './workspace/index-manager';
 import type { WorkspaceManager } from './workspace/workspace-manager';
 import type { LanguageServer } from './lsp/language-server';
+import { SignatureHelpProvider } from './lsp/signature-help-provider';
```

Comment:
Minor: We only import the type here to prevent cyclic dependencies
```suggestion
import type { SignatureHelpProvider } from './lsp/signature-help-provider';
```